.github/workflows/nix.yml

```
@@ -67,10 +67,12 @@ jobs:
 
     name: wash-${{ matrix.config.target }}
     runs-on: macos-12
+    # NOTE(vados-cosmonic): Mac builds can OOM in GitHub CI
+    continue-on-error: true
```

The only place where I have seen this fail was the install step, exactly once. Rather than silently ignoring errors here and completely breaking the assumptions of this CI check, the issue should be fixed in `build-nix` as was discussed offline - use plain bundled `install` binary rather than using GNU coreutils one.
If you don't want to do this in this PR, then please file an issue, but IMO, the proposed change is not a solution for the problem

---

And like we discussed offline, this is actually more complicated because `nix` did provide us a way to be cross-platform (in particular, we could ignore windows' lack of `install`) -- if you have some confidence that the error was only temporary/unlikely to reoccur, then I'll re-enable this step!

if not, I'd prefer to file an issue and have someone who is more experienced with `nix` take a look, and we can debug that way.

That said, there's another option here -- I'm going to mark `continue-on-error` only for *non* release branches. This way regular work can continue, but during a release, it has to pass.